src/screens/MainApp/MyDiagnoses/index.js

```diff
@@ -40,35 +45,55 @@ const MyDiagnoses = () => {
       setDiagnoses(null)
     } finally {
       setDownloadingDiagnoses(false)
+      setRefreshing(false)
     }
   }
 
   useEffect(() => {
     getRenderedDiagnoses()
-  }, [refetchTrigger])
+  }, [refetch])
 
   if (!isNull(diagnoses) && isEmpty(diagnoses)) {
     return <NoDiagnoses />
   }
 
   return (
-    <View style={styles.container}>
-      <ForceRerenderOnNavigation resetStateFunction={refetchDiagnoses} />
-      {downloadingDiagnoses && <ActivityIndicator size='large' />}
-      {diagnoses &&
-        <FlatList
-          data={diagnoses}
-          showsVerticalScrollIndicator={false}
-          renderItem={({ item }) => item}
-          keyExtractor={item => String(diagnoses.indexOf(item))}
-        />}
-    </View>
+    <>
+      <View style={styles.container}>
```

Comment:
There's no need for the fragment here (`<>`). Everything is wrapped by the `View` component.